lib/core/versions/latest/ValueTimeLockVerifier.ts

```diff
@@ -43,12 +48,14 @@ export default class ValueTimeLockVerifier {
    * @param numberOfOperations The target number of operations.
    * @param sidetreeTransactionTime The transaction time where the operations were written.
    * @param sidetreeTransactionWriter The writer of the transaction.
+   * @param versionMetadataMapper The mapper from transaction time to version metadata
    */
   public static verifyLockAmountAndThrowOnError (
     valueTimeLock: ValueTimeLockModel | undefined,
     numberOfOperations: number,
     sidetreeTransactionTime: number,
-    sidetreeTransactionWriter: string): void {
+    sidetreeTransactionWriter: string,
+    versionMetadataMapper: IVersionMetadataMapper): void {
 
     // If the number of written operations were under the free limit then there's nothing to check
     if (numberOfOperations <= ProtocolParameters.maxNumberOfOperationsForNoValueTimeLock) {

```

Comment:
Sanity: do we need to expose this to `AbstractVersionMetadata` also?

Reply:
After discussion concluded that we don't.